docs/source/markdown/podman-secret-ls.1.md.in

```diff
@@ -28,6 +28,20 @@ Valid filters are listed below:
 
 Format secret output using Go template.
 
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder**     | **Description**                                                        |
+| ------------------- | ------===================================----------------------------- |
```

Comment:
EEK! Those equal signs make a complete mess of the table.

Reply:
Dan, don't do stuff while running out to lunch.